ridgesecurity-ridgebot/info.json

```diff
@@ -0,0 +1,135 @@
+{
+  "name": "ridgebot",
+  "label": "RidgeBot",
+  "description": "RidgeBot",
+  "publisher": "Ridge Security",
+  "cs_approved": true,
+  "cs_compatible": true,
+  "version": "1.0.0",
+  "category": "Penetration Testing",
+  "help_online": "https://ridgesecurity.ai/resources/",
+  "icon_small_name": "ridge_small.png",
+  "icon_large_name": "ridge_large.png",
+  "configuration": {
+      "fields": [
+          {
+              "title": "Server URL",
+              "name": "server_url",
+              "required": true,
+              "editable": true,
+              "visible": true,
+              "type": "text"
+          },
+          {
+              "title": "User Token",
+              "name": "user_token",
+              "required": true,
+              "editable": true,
+              "visible": true,
+              "type": "password"
+          }
+      ]
+  },
+  "operations": [
+      {
+          "operation": "get_task_info",
+          "title": "get_task_info",
+          "description": "Get Task Info List for Single Task",
+          "enabled": true,
+          "parameters": [
+              {
+                  "title": "Task ID",
+                  "required": true,
+                  "visible": true,
+                  "editable": true,
+                  "type": "text",
+                  "name": "task_id"
```

Comment:
please add descriptions of all parameters. this will helps us to build the doc for connector.

Reply:
hi @cs-prerna-joshi, I have added the descriptions on all actions and parameters as mentioned.